<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>激光点云数据三维可视化</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKOpenGLWidget" name="Map" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>1190</width>
      <height>530</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="propertyTable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>256</width>
      <height>531</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>180</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AllEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>8</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Property</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>test</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>test</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>test</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>test</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>test</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>test</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>test</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>test</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>569</y>
      <width>1501</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainChartLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选项</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);
</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionactionRotateRight"/>
   <addaction name="actionRotateLeft"/>
   <addaction name="actionMoveLeft"/>
   <addaction name="actionMoveRight"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionOpenOceanFile"/>
   <addaction name="actionOceanStart"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionactionRotateRight">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/images/images/rotate_right.png</normaloff>:/images/images/rotate_right.png</iconset>
   </property>
   <property name="text">
    <string>actionRotateRight</string>
   </property>
   <property name="toolTip">
    <string>RotateRight</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/images/images/rotate_left.png</normaloff>:/images/images/rotate_left.png</iconset>
   </property>
   <property name="text">
    <string>actionRotateLeft</string>
   </property>
   <property name="toolTip">
    <string>RotateLeft</string>
   </property>
  </action>
  <action name="actionMoveRight">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/images/images/right.png</normaloff>:/images/images/right.png</iconset>
   </property>
   <property name="text">
    <string>actionMoveRight</string>
   </property>
   <property name="toolTip">
    <string>MoveRight</string>
   </property>
  </action>
  <action name="actionMoveLeft">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/images/images/left.png</normaloff>:/images/images/left.png</iconset>
   </property>
   <property name="text">
    <string>actionMoveLeft</string>
   </property>
   <property name="toolTip">
    <string>MoveLeft</string>
   </property>
  </action>
  <action name="actionOpenOceanFile">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>OpenOceanFile</string>
   </property>
   <property name="toolTip">
    <string>OpenOceanFile</string>
   </property>
  </action>
  <action name="actionOceanStart">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/images/images/play.png</normaloff>:/images/images/play.png</iconset>
   </property>
   <property name="text">
    <string>OceanChartStart</string>
   </property>
   <property name="toolTip">
    <string>OceanChartStart</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
